dask/array/routines.py

```diff
@@ -1843,6 +1843,22 @@ def piecewise(x, condlist, funclist, *args, **kw):
     )
 
 
+@derived_from(np)
+def select(condlist, choicelist, default=0):
+    return blockwise(
+        np.select,
+        "m",
```

Comment:
Given the suggestion below, this would then become:
```suggestion
        "tuple(range(1, choicelist[0].ndim + 1)),",
```